roles/privacyidea/templates/privacyideaapp.py.j2

```diff
@@ -0,0 +1,5 @@
+import sys
+sys.stdout = sys.stderr
+from privacyidea.app import create_app
+application = create_app(config_name="production", config_file="/etc/privacyidea/pi.cfg")
+application.debug = True
```

Comment:
should this be true for all stages?

Reply:
Since the application should not be run in debug mode in production , i remove debug=False

Reply:
can variables be used in that file? Introducing a var could help?

Reply:
i think it is good to have the option to increase the loglevel. If one could use a var that yould be awesome